### ISSUE TITLE
No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"

### ISSUE THREAD

**Issue by afternoon2** [software-mansion/TypeGPU#703]
When using `typegpu @0.2.1` with Bun, I get the following error when running the install script:

```
[ERROR] No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"

    node_modules/typegpu/chunk-5EPBCOO4.js:1:1221:
      1 │ ...Object expected");y(h)}),1)}}),a[ue("iterator")]=()=>a,r("next"),"throw"in t?r("throw"):a.throw=o=>{throw o},"return"in t&&r("return"),a};import{MaxValue as Qe,Measurer as Xe,Schema as Ye}from"typed-binary";var Re=class e extends Error{constructor(n,r){let a=r.map(o=>`- ${o}`);a.length>20&&(a=[...a.slice(0,11),"...",...a.slice(-10)]);super(`Resolution o...
        ╵                                                                                                                                                                                   ~~~~~~

✘ [ERROR] No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"

    node_modules/typegpu/chunk-5EPBCOO4.js:3:2297:
      3 │ ...(2147483647,t))},q=Object.assign(et,ye(w.i32,"i32")),tt=e=>z()?`f32(${e})`:typeof e=="boolean"?e?1:0:e,J=Object.assign(tt,ye(w.f32,"f32"));import{MaxValue as v,Measurer as je,Schema as We}from"typed-binary";function Q(){return new me}var me=class{constructor(){s(this,"label")}$name(t){return this.label=t,this}resolve(t){return t.nameFor(this)}toString()...
        ╵                                                                                                                                                                                   ~~~~~~

✘ [ERROR] No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"

    node_modules/typegpu/chunk-5EPBCOO4.js:3:4825:
      3 │ ...:"<unnamed>"}=${n}]`}};function rt(e,t){let n="size"in e?e.size:e.currentByteOffset,r=t-1,a=n&r;"skipBytes"in e?e.skipBytes(t-a&r):e.add(t-a&r)}var l=rt;import{Measurer as $e,Schema as Ne}from"typed-binary";var B=(e,t)=>{let n=t-1,r=~n;return e&n?(e&r)+t:e};import{MaxValue as at,Measurer as ot}from"typed-binary";function be(e,t){return fe(e)?new M(e.inn...
        ╵                                                                                                                                                                                   ~~~~~~

✘ [ERROR] No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"

    node_modules/typegpu/chunk-5EPBCOO4.js:7:9634:
      7 │ ...pe:T,rows:4,columns:4,makeFromColumnVectors(...[e,t,n,r]){return new ae(...e,...t,...n,...r)},makeFromElements:(...e)=>new ae(...e)});import{BufferReader as wt,Measurer as gt,Schema as ht}from"typed-binary";var u=class extends ht{constructor(n,r){var o;super();this.size=n;this.kind=r;s(this,"underlyingType");s(this,"elementSize");s(this,"elementCount");...
        ╵                                                                                                                                                                                   ~~~~~~

✘ [ERROR] No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"

    node_modules/typegpu/chunk-5EPBCOO4.js:7:13357:
      7 │ ...,Sr=new u(8,"sint32x2"),kr=new u(12,"sint32x3"),Vr=new u(16,"sint32x4"),Ir=new u(4,"unorm10_10_10_2");function kt(e){return{scope:"function",pointsTo:e}}import{Measurer as Vt,Schema as It}from"typed-binary";function Pt(e){return new oe(e)}function _t(e){return e instanceof oe}var oe=class extends It{constructor(n){super();this.innerData=n;s(this,"size")...

Error: Build failed with 5 errors:
node_modules/typegpu/chunk-5EPBCOO4.js:1:1221: ERROR: No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"
node_modules/typegpu/chunk-5EPBCOO4.js:3:2297: ERROR: No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"
node_modules/typegpu/chunk-5EPBCOO4.js:3:4825: ERROR: No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"
node_modules/typegpu/chunk-5EPBCOO4.js:7:9634: ERROR: No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"
node_modules/typegpu/chunk-5EPBCOO4.js:7:13357: ERROR: No matching export in "node_modules/typed-binary/dist/index.js" for import "Schema"
    at failureErrorWithLog (/Users/(...)/node_modules/esbuild/lib/main.js:1476:15)


Node.js v20.9.0
error: script "dev" exited with code 1
```

**Comment by iwoplaza** [comment]
This was a regression introduced in `typed-binary@3.4.0`, I upped the minimum version we depend on to `3.4.1`, which fixed the regression. I released `typegpu@0.2.2` with this fix, let me know if this fixes the issue for you! 🙌

**Comment by afternoon2** [comment]
it works, thank you
